api/routes/utils.js

```diff
@@ -710,26 +752,50 @@ var utils = {
     return cb(null);
   },
 
-  // Given a category name, update the percentiles and percentages, for all investors
-  updateInvestors: function(category, cb, username, expertCategoryTotal) {
+  // Given a category name, update the percentiles for all the experts and investors in that category
+  // In addition, update the investor dividends
+  updatePercentilesAndDividends: function(categoryName, expert, cb) {
     var self = this;
-    var investorsPromise = User.findInvestorByCategory(category, function() {});
+    var i = self.getCategoryIndex(expert, categoryName);
+    if (i === -1) {
```

Comment:
Can this just be ```if i < 0```? If you don't want to change it it's fine. Just seems like a safer convention to use in general.